### PR TITLE
[FEATURE][SWE-1334] Wrote components Title, Text

### PR DESCRIPTION
**Pull request by Olexandr25**

### CODE REVIEW

File: src/components/elements/Text/Text.styled.js [hunk 1]
```diff
@@ -1,7 +1,104 @@
...
+    line-height: var(--brandbook-module-text-body1-line-height);
+    font-family: var(--brandbook-module-text-body1-font-family);
+  `,
+  body2: css`
+    font-size: var(--brandbook-module-text-body2-font-size);
+    line-height: var(--brandbook-module-text-body2-line-height);
+    font-family: var(--brandbook-module-text-body2-font-family);
+  `,
+  // Brand text
+  alphabet: css`
```
Comment (olenashevchuk) [inline src/components/elements/Text/Text.styled.js:26]: add "brand" prefix

File: src/components/elements/Title/Title.styled.js [hunk 2]
```diff
@@ -1,7 +1,122 @@
...
+    line-height: var(--brandbook-module-title-cover-line-height);
+    letter-spacing: var(--brandbook-module-title-cover-letter-spacing);
+    font-family: var(--brandbook-module-title-cover-font-family);
+  `,
+  greetings: css`
+    font-size: var(--brandbook-module-title-greetings-font-size);
+    line-height: var(--brandbook-module-title-greetings-line-height);
+    font-family: var(--brandbook-module-title-greetings-font-family);
+  `,
+  title: css`
```
Comment (olenashevchuk) [inline src/components/elements/Title/Title.styled.js:16]: Do we not have h1-h6 sizes for presentation's titles?

Comment (Olexandr25) [reply src/components/elements/Title/Title.styled.js:16]: I left for them previous naming, only for this part
Xl,lg,md,sm,xs 
I can change naming for this part

Comment (Olexandr25) [reply src/components/elements/Title/Title.styled.js:16]: Screenshot 2022-07-25 at 08 51 50

**Review by olenashevchuk — APPROVED** [review]

**Pull request merged**